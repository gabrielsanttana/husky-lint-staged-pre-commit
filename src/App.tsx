import "./App.css";

const App: React.FC = () => {
  return (
    <div>
      <h1>Husky</h1>
    </div>
  );
};

export default App;
