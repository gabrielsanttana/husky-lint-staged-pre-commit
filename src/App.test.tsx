import {render} from '@testing-library/react';
import App from './App';

describe('The App', () => {
  it('renders correctly', () => {
    const {container} = render(<App />);

    expect(container).toMatchSnapshot();
  });
});
