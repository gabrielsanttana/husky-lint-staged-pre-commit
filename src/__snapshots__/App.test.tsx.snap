// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The App renders correctly 1`] = `
<div>
  <div>
    <h1>
      Husky
    </h1>
  </div>
</div>
`;
